makeflow/README.md

```diff
@@ -0,0 +1,23 @@
+# Agriculture Processing Pipeline: Makeflow
```

Comment:
Would it be possible to provide a small dataset that could be used to run this? It could be available on Google Drive under the account that Jorge setup in .tar format?

Reply:
Yes it's possible.

Or we could change this to grab files from the Data Store where Eric put the data for the class.

Reply:
I don't think the class has a small dataset

Reply:
New issue: A small dataset to test pipeline #49